src/libsyncengine/update_detection/update_detector/updatetreeworker.cpp

```diff
@@ -1187,6 +1174,96 @@ ExitCode UpdateTreeWorker::updateNodeWithDb(const std::shared_ptr<Node> parentNo
     return ExitCodeOk;
 }
 
+ExitCode UpdateTreeWorker::updateTmpNode(const std::shared_ptr<Node> tmpNode) {
+    SyncPath dbPath;
+    // If it's a move, we need the previous path to be able to retrieve database data from path
```

Comment:
Is it correct that here we look for the source path of the move operation?

Reply:
Yes because we will retrieve information about this item in DB based on its path. Since the content of the DB is the image of the last succeful synchronisation, we need the source path to retrieve the moved item.